packages/apis/src/btc_staking_api.rs

```diff
@@ -180,10 +180,10 @@ pub struct ActiveBtcDelegation {
     pub end_height: u32,
     /// Total BTC stakes in this delegation, quantified in satoshi
     pub total_sat: u64,
-    /// Staking tx.
-    pub staking_tx: Binary,
-    /// Slashing tx.
-    pub slashing_tx: Binary,
+    /// Staking tx as hex string.
+    pub staking_tx_hex: String,
+    /// Slashing tx as hex string.
+    pub slashing_tx_hex: String,
```

Comment:
I don't think this struct should be changed due to the API because it is mostly an internal DB object.

Instead I think we should define dedicated structs for APIs

Reply:
It is an intermediate struct converted from the [IBC packet](https://github.com/babylonlabs-io/cosmos-bsn-contracts/blob/b298a4d92e8301167eea8d7093d313a691107886/contracts/babylon/src/ibc.rs#L185-L213) (which is part of IBC module's DB). The object corresponding to this struct is used for a series of verifications [here](https://github.com/babylonlabs-io/cosmos-bsn-contracts/blob/b298a4d92e8301167eea8d7093d313a691107886/contracts/btc-staking/src/staking.rs#L24-L34).

So I think a proper way of fixing this is to create a new struct `BTCDelegationResponse` that can be converted from and to the [BtcDelegation](https://github.com/babylonlabs-io/cosmos-bsn-contracts/blob/b298a4d92e8301167eea8d7093d313a691107886/contracts/btc-staking/src/state/staking.rs#L71-L110) DB struct